src/mantine-core/src/components/AvatarGroup/AvatarsGroup.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import cx from 'clsx';
+import { Avatar, Text } from '@mantine/core';
+import { DefaultProps, MantineNumberSize, useMantineTheme, mergeStyles } from '../../theme';
+import useStyles from './AvatarsGroup.styles';
+
+export type AvatarsGroupStylesNames = keyof ReturnType<typeof useStyles>;
+
+export interface AvatarsGroupProps extends
```

Comment:
Also add `spacing` to handle spacing between avatars, by default it should be negative so that avatars will overlay each other